### DISCUSSION TITLE
Max pattern length

### DISCUSSION THREAD

**Discussion by simongiles1** [krisk/Fuse#768] [Q&A] [answered]
Hi, I noticed that maxPattern used to be an option (I didn't check multiple versions but it's available in v3.4.5). It looks like this feature has been removed. Is there an alternate feature that I can use to achieve this same functionality? I have looked through the docs and can't seem to find anything.
Thanks in advance.

**Comment by krisk** [comment, marked as answer]
Sorry for the late reply.

The `maxPatternLength` option was removed because it's no longer needed. The Bitap algorithm has a 32-bit limit per chunk, but Fuse now automatically splits longer patterns into 32-character chunks and merges the results. So there's no hard cap on pattern length anymore — it just works.
